addons/auth_signup/views/res_config_settings_views.xml

```diff
@@ -18,7 +18,7 @@
                                 Let your customers log in to see their documents
                             </div>
                             <div class="mt8">
-                                <field name="auth_signup_uninvited" class="o_light_label" widget="radio" options="{'horizontal': true}" required="True"/>
+                                <field name="auth_signup_uninvited" class="o_light_label" widget="radio" options="{'horizontal': true}"/>
```

Comment:
Thinking out loud but shouldn't we leave that `required` field and just override the value in a xpath next to here 
https://github.com/odoo/odoo/pull/89707/files#diff-bf2411ed72ba8f2d9a6176188decb4c873ea36565df56858ee69e4387c6aab89R10 ?

That would preserve the original required from auth_signup and prevent it to raise the warning on save if website is installed -> Website hides it and makes it required only if a website is selected

What do you think

Reply:
We could, but we still have an issue if we have website and sale installed without `website_sale` as it also shows up in the sales page.
Fortunately for us, it has a default value and it being a radio widget, there is no way from the interface to set it to a falsy value.

We can also keep the default value when website is installed.
As it will simply do nothing if no website is selected.